Shared/Samples/Show device location/ShowDeviceLocationView.swift

```diff
@@ -42,10 +65,10 @@ struct ShowDeviceLocationView: View {
                             }
                         }
                     }
-                    .disabled(model.areSettingsDisabled)
+                    .disabled(settingsButtonIsDisabled)
```

Comment:
I'm not sure this is working as intended, or even needed. The button seems to dim itself automatically when the menu is presented.

Reply:
I think this was intended for failing to create the location data source. Say the user denied location permission for the app, or some other reasons that prevent the LDS from starting. Unfortunately Simulator has a [bug](https://developer.apple.com/forums/thread/693317) right now that doesn't allow us to test it.

Reply:
Ah okay, I see that it's performing as intended then on my physical iPad. Thanks!